docs/install/docker.md

```diff
@@ -41,8 +41,61 @@ of the image to derive the container from.
 The Percona Server for MongoDB container exposes standard MongoDB port (27017),
 which can be used for connection from an application
 running in another container.
-To link the application container to the `psmdb` container,
-use the `--link psmdb` option when running the container with your app.
+
+To connect the application container to the `psmdb` container, you can either use the default `bridge` network, or create your own network and attach both containers to it.
+
+=== "Bridge network"
+    
+    When you start Docker, a default `bridge` network is created and all containers are attached to it unless otherwise specified. 
+
+    To link your application container to the `psmdb` container, do the following:
+
+    1. Get the IP address of the `psmdb` container
+
+        ```{.bash data-prompt="$"}
+        $ docker inspect psmdb | grep "IPAddress"
```

Comment:
Better to use
`docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' psmdb`
and the result just internal ip, e.g:
`172.29.0.4`

Reply:
Updated